src/escpos/escpos.py

```diff
@@ -733,6 +735,18 @@ def panel_buttons(self, enable=True):
         else:
             self._raw(PANEL_BUTTON_OFF)
 
+    def queryStatus(self):
+        self._raw(RT_STATUS_ONLINE)
+        time.sleep(1)
+        status = self._read();
+        if (len(status) > 0):
+            return status;
+        else:
+            return [8];
+
+    def _isOnline(self):
```

Comment:
This has been my first-time Python coding - wasn't sure about the convention.
Thanks!